packages/data/README.md

```diff
@@ -712,6 +712,24 @@ _Returns_
 
 -   `Function`: A custom react hook.
 
+<a name="useStoreSelectors" href="#useStoreSelectors">#</a> **useStoreSelectors**
+
+A shorthand for useSelect() that returns all the selectors from a given store.
```

Comment:
`useSelect` is explained as:

> Custom react hook for retrieving props from registered selectors.

Documentation should follow the same theme. Doesn't it rather returns data processed by selectors from a single store.

Reply:
A usage example would be very helpful as well.

One more, why did you pick the ending `Selectors` rather the `Select` that the original hook uses?

Reply:
> One more, why did you pick the ending Selectors rather the Select that the original hook uses?

The original goes like this: `useSelect( (select) => select('store').selector() )` - what you get is the `select` function, hence the name `useSelect` makes sense.

This hook goes like: `useStoreSelectors( ({selector}) => selector() )` - what you get are all the selectors from a given store, hence the name `useStoreSelectors` makes more sense than `useStoreSelect`.

Reply:
Yes, it sounds convincing. 👍 

Aside: I'm wondering if @nerrad had the same mindset when proposing `useSelect` :)

Reply:
As a matter of fact: #15512